logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -48,7 +48,8 @@ data class Conversation(
     val accessRole: List<AccessRole>,
     val creatorId: String?,
     val receiptMode: ReceiptMode,
-    val messageTimer: Duration? // enforced time by group conversation for self deleting messages, always NULL for 1on1 conversation
+    val messageTimer: Duration?, // enforced time by group conversation for self deleting messages, always NULL for 1on1 conversation
+    val userMessageTimer: Duration? // timer duration picked by the user in absence of team or conversation enforced timers
```

Comment:
Kdoc would be great here. So whoever presses F1 or Ctrl+Q on the IDE can see what's this about.